Makefile

```diff
@@ -129,4 +129,4 @@ production: ## Set environment to production
 
 .PHONY: check-if-migrations-to-run
 check-if-migrations-to-run:
-	@echo $(shell API_HOST_NAME=https://api.${DNS_NAME} python3 scripts/check_if_new_migration.py)
+	@echo $(shell python3 scripts/check_if_new_migration.py)
```

Comment:
We're currently setting `API_HOST_NAME ` [in the pipelines](https://github.com/alphagov/notifications-aws/blob/main/concourse/api-pipeline.yml.j2#L231). The preview, staging and production targets aren't actually used anywhere now, so could be deleted.

Would you prefer the pipelines to be changed to set `DNS_NAME` instead of `API_HOST_NAME`?

Reply:
Ok, I've removed `preview`, `staging` and `production`. It was only recently that `check-if-migrations-to-run` was changed to add in `API_HOST_NAME` (which broke things), so this change just puts things back to how they were a few weeks ago.